tests/foreman/ui/test_oscap_tailoringfiles.py

```diff
@@ -94,39 +110,26 @@ def test_positive_create_with_space(self):
             2. Upload a valid tailoring file
             3. Give it a name with space
 
-        :CaseAutomation: notautomated
-
-        :expectedresults: Tailoring file will be added to satellite
-
-        :CaseImportance: Critical
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_positive_create_with_valid_file(self):
```

Comment:
why this test has been removed

Reply:
All positive tests , uses valid tailoring file , this was a redundant test. So , removed